packages/lib/services/e2ee/EncryptionService.ts

```diff
@@ -123,41 +130,78 @@ export default class EncryptionService {
 	}
 
 	public isMasterKeyLoaded(masterKey: MasterKeyEntity) {
-		const d = this.decryptedMasterKeys_[masterKey.id];
+		if (this.encryptedMasterKeys_.get(masterKey.id)) {
+			return true;
+		}
+		const d = this.decryptedMasterKeys_.get(masterKey.id);
```

Comment:
With this change, a master key is "loaded" if it is registered with the `EncryptionService` (and hasn't **yet** failed to decrypt). Previously, master keys were only registered with the `EncryptionService` after they had decrypted successfully.